route/matcher.go

```diff
@@ -12,6 +12,7 @@ type matcher func(uri string, r *Route) bool
 var Matcher = map[string]matcher{
 	"prefix": prefixMatcher,
 	"glob":   globMatcher,
+	"noCase": noCaseMatcher,
```

Comment:
I feel like this might be a terrible name, but it's the best I could come up with this evening. I'm open to suggestions

Reply:
Maybe `nocase`